extension/data_loader/targets.bzl

```
@@ -54,7 +54,16 @@ def define_common_targets():
 
     runtime.cxx_library(
         name = "mmap_data_loader",
-        srcs = ["mmap_data_loader.cpp"],
+        srcs = [
+            "mmap_data_loader.cpp",
+            "mman_windows.cpp"
+        ] if host_info().os.is_windows else [
+            "mmap_data_loader.cpp"
+        ],
```

Something like this would avoid repeating the common part.

```suggestion
        srcs = ["mmap_data_loader.cpp"] + 
            ["mman_windows.cpp"] if host_info().os.is_windows else [],
```